{"name": "Проверка ошибки создания курьера: создание двух курьеров с одинаковым логином", "status": "passed", "start": 1699616037445, "stop": 1699616037835, "uuid": "80ab15d6-a474-4fdd-b423-6d179a34780d", "historyId": "1e791bea568bd8e2f9dd08c52c2ba495", "testCaseId": "1e791bea568bd8e2f9dd08c52c2ba495", "fullName": "tests.test_courier_creation.TestCourierCreation#test_courier_duplicate_login_failed_create", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_creation"}, {"name": "subSuite", "value": "TestCourierCreation"}, {"name": "host", "value": "WKS-084"}, {"name": "thread", "value": "20968-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_creation"}]}